FIAT/arnold_qin.py

```diff
@@ -0,0 +1,66 @@
+# This file is part of FIAT (https://www.fenicsproject.org)
+#
+# SPDX-License-Identifier:    LGPL-3.0-or-later
+#
+# Written by Pablo D. Brubeck (brubeck@protonmail.com), 2024
+
+from FIAT import finite_element, polynomial_set
+from FIAT.bernardi_raugel import BernardiRaugelDualSet
+from FIAT.quadrature_schemes import create_quadrature
+from FIAT.reference_element import TRIANGLE
+from FIAT.macro import CkPolynomialSet
+from FIAT.hct import HsiehCloughTocher
+
+import numpy
+
+
+def ArnoldQinSpace(ref_el, degree, reduced=False):
```

Comment:
Not sure if what values for `degree` makes sense here. Won't the P0_x (eventually) be redundant for higher degree?

Reply:
I moved the checks for `degree == 2` and `cell == triangle` to the function space. We would fail to define the element if we fail to define the space.